src/Orckestra.Media.AutoImageResizing/Plugin.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Composite.Core.WebClient.Renderings.Page;
+using Orckestra.Composer;
+
+namespace Orckestra.Media.AutoImageResizing
+{
+    public class Plugin : IComposerPlugin
```

Comment:
Instead of IComposerPlugin, please use [ApplicationStartup] attribute instead. 
Image resizing functionality can work even without RefApp, so there's no need for it to reference RefApp/OCC DLLs at all. 
Only reference Composite.dll (as done already for IPageContentFilter), and use the startup handler example from here:

https://docs.c1.orckestra.com/Layout/Filtering-Page-Output